src/mobot_client/timeouts.py

```diff
@@ -77,7 +77,7 @@ def do_refund(self, customer):
         # FIXME TODO
 
         # Get amount customer paid
-        price = session.drop.item.price_in_pmob
+        price = session.drop.item.price_in_mob
 
         self.payments.send_mob_to_customer(customer, customer.phone_number, mc.pmob2mob(price), True)
```

Comment:
should this be mc.pmob2mob(price) or just price? It looks like this PR changes price to be in mob rather than pmob

Reply:
Yeah, we can totally just do "price". The other option is to eventually switch to an actual "Currency" field type, which I've used before and gives us things like conversions for MOB to USD/GBP. It's also a decimal-backed field.